agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -2197,6 +2198,16 @@ bool backhaul_manager::handle_1905_autoconfiguration_response(ieee1905_1::CmduMe
         return true;
     }
 
+    auto beerocks_header = message_com::parse_intel_vs_message(cmdu_rx);
+    if (beerocks_header &&
+        beerocks_header->action_op() == beerocks_message::ACTION_CONTROL_SLAVE_HANDSHAKE_RESPONSE) {
+        // mark controller as prplMesh
+        LOG(DEBUG) << "prplMesh controller: received ACTION_CONTROL_SLAVE_HANDSHAKE_RESPONSE from "
+                   << src_mac;
+        prplmesh_controller = true;
+    } else {
+        LOG(DEBUG) << "Not prplMesh controller " << src_mac;
```

Comment:
Since we get multiple responses in the backhaul manager, perhaps we should log an error message if we already had prplmesh_controller == true. And we should probably reset it to false as well.